modules/ROOT/pages/8.0-release-notes.adoc

```diff
@@ -126,6 +138,11 @@ For information on using Enhanced Skins & Icon Packs, see: xref:enhanced-skins-a
 
 // CCFR here.
 
+=== Split button DOM structure changes
+// #TINY-8665
+
+The split buttons are now rendered as two separate buttons: action and dropdown. Each has its own event handling and focus management. Split button components now use `+.tox-split-button__main+` and `+.tox-split-button__chevron+` CSS classes. Custom CSS that targets split button internals may need to be updated to reflect these changes.
```

Comment:
Not sure if a separate changelog entry is necessary. This could be the second paragraph of the `Split buttons are now rendered as separate action and chevron buttons`.